department-of-veterans-affairs/va.gov-cms
# VACMS-1201: Changing perms to disallow listing creation.

## Description
See #1201 

## Testing done
 - Visual

## QA steps
 - As a user with content admin role, go to `/node/add` and visually verify that you do not see any listing node types.
